backend/db/migrate/20220813043556_create_events.rb

```diff
@@ -0,0 +1,11 @@
+class CreateEvents < ActiveRecord::Migration[7.0]
+  def change
+    create_table :events, id: :string do |t|
```

Comment:
Does ruby support uuids? I do recommend using them as db storage for PKs for scalability purposes - though if we were super serious about scaling, we might use a no-sql db. As it is, pg gets the job done fine

Reply:
It does, and I was thinking about using those, but I decided on `HumanToken` instead for a couple of reasons:

1. These IDs will appear in URLs (including permalinks) and cookies, so I thought something more accessible would be useful;
2. HumanToken should provide enough uniqueness for our purposes.

That said, thinking on it a bit more, I wouldn't be opposed to switching to UUIDs, and Rails does support them out of the box with `SecureRandom.uuid`. Yeah, I think I may switch us over.